Makefile

```diff
@@ -90,6 +90,7 @@ endif
 
 .PHONY: docker-tests
 docker-tests:
+	export OPENQA_LOGFILE=openqa-debug.log ;\
```

Comment:
this needs to be:

`export OPENQA_LOGFILE=/opt/openqa/openqa-debug.log` 

Otherwise travis is unable to access it.